viewer/packages/rendering/src/glsl/base/updateFragmentDepth.glsl

```diff
@@ -11,20 +11,9 @@ float computeFragmentDepth(vec3 p, mat4 projectionMatrix) {
   return ((gl_DepthRange.diff * (projected_intersection_z / projected_intersection_w)) + gl_DepthRange.near + gl_DepthRange.far) * .5;
 }
 
-#if defined(gl_FragDepthEXT) || defined(GL_EXT_frag_depth) 
-
-float updateFragmentDepth(vec3 p,mat4 projectionMatrix) {
-  gl_FragDepthEXT = computeFragmentDepth(p, projectionMatrix);
-  return gl_FragDepthEXT;
-}
```

Comment:
Shouldn't this be kept but updated to use new mechanism for frag depth?

Or was it unused?

Reply:
It is using new mechanism for frag depth